Buildings/Resources/Documentation/userGuide/build/html/_sources/development.rst.txt

```diff
@@ -619,7 +619,7 @@ The following rules need to be followed, in addition to the guidelines described
 
 #. The size of the icon should be such that it provides a good fit for all the input and output interfaces. The minimum
    recommended icon size is 100 by a 100. If there are many interfaces the icon size should be extended in vertical direction.
-   Icons should be symetrical with reference to the grid origin. E.g, the default specification is
+   Icons should be symmetrical with reference to the grid origin. E.g, the default specification is
```

Comment:
ups, I forgot the spell check isn't active as I've changed text editors. Apologies!